preview/src/main.rs

```diff
@@ -223,6 +240,113 @@ fn SwitchExample() -> Element {
     }
 }
 
+#[component]
+fn ToastExample() -> Element {
+    // Get the toast API
+    let toast_api = use_toast();
```

Comment:
I'm not certain on the API here - might revisit it in the future, but it is good for now.